justega247/favorite-things
# #167510764: Users can create favorite things

#### What does this PR do?
- This PR gives users of the application an avenue to add favorite things to the app
#### Description of Task to be completed?
- write the code for the implementation
- write the code to implement the design
#### How should this be manually tested?
- On your local version of this project
- Startup the backend server using **python manage.py runserver**
- Navigate into the client directory and start-up webpack using **npm start**
- Click on the `Get started` button
- Click on the `Add Favorite` button
- You should see a form to enter the details needed
- On supplying valid data and clicking on the Create Favorite button, then the favorite should be created.
#### What are the relevant pivotal tracker stories?
[#167510764](https://www.pivotaltracker.com/story/show/167510764)
